diff --git a/src/cpp/plantgl/algo/base/pointmanipulation.cpp b/src/cpp/plantgl/algo/base/pointmanipulation.cpp
index 3dbd070..127eb23 100644
--- a/src/cpp/plantgl/algo/base/pointmanipulation.cpp
+++ b/src/cpp/plantgl/algo/base/pointmanipulation.cpp
@@ -383,6 +383,7 @@ public:
 
 	void main(Index const &path, RealArrayPtr const &radii, boost::asio::thread_pool &threadPool)
 	{
+#ifdef PGL_WITH_ANN
 		uint_t index = 0;
 		KDTree3 kdtree(m_points);
 
@@ -408,6 +409,7 @@ public:
 			boost::asio::post(threadPool, boost::bind(&ThreadAdapterWireSelection::operator(), this, addWire(), boost::ref(closestPoints[index]), center, vector, height));
 			++progress;
 		}
+#endif // PGL_WITH_ANN
 	}
 
 	void operator()(boost::shared_ptr<Index> wire, Index const &closestPoints, Vector3 const &center, Vector3 const &vector, real_t height)
